molo/profiles/templates/profiles/change_password.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n %}
 
 {% block content %}
-<div class="form-wrap change-password">
+<div class="profiles l-indent">
```

Comment:
`l-indent` won't scale to other websites, so we'll have to add padding to the `profiles` class.